### PR TITLE
replace ng-datetime-picker package dependency

### PR DESCRIPTION
**Pull request by Yagnik56**
This PR replace ng-datetime-picker package dependency with @danielmoncada/angular-datetime-picker.

### CODE REVIEW

**Comment by Yagnik56** [comment]
Hey @danoswaltCL, before you approve this PR it will be good if you check this changes locally to confirm it is working fine. I checked it from my side but it will be good if u can do it as well.

**Review by danoswaltCL — APPROVED** [review]

**Pull request merged**
